tools/importer-rest-api-specs/components/parser/models.go

```diff
@@ -509,7 +515,7 @@ func (d SwaggerDefinition) parseObjectDefinition(modelName, propertyName string,
 		}
 
 		// then call ourselves to work out what to do with it
-		objectDefinition, nestedResult, err := d.parseObjectDefinition(*objectName, propertyName, topLevelObject, knownIncludingPlaceholder)
+		objectDefinition, nestedResult, err := d.parseObjectDefinition(*objectName, propertyName, topLevelObject, knownIncludingPlaceholder, true)
```

Comment:
presumably this should be passing `parsingModel` in when looping around? else we'll potentially be passing `true` when the `false` thing:

```suggestion
		objectDefinition, nestedResult, err := d.parseObjectDefinition(*objectName, propertyName, topLevelObject, knownIncludingPlaceholder, parsingModel)
```

Reply:
I think we should always pass a `true` here, because the Schame represented by `topLevelObject` is what the `objectName` want. that means we can use `topLevelObject` to parse model `objectName`. also as other two calls below, the inner/inheritedModel is corresponding to the modelName, and they should always pass a `true`.